chain/indexer/src/block.rs

```diff
@@ -55,20 +57,23 @@ lazy_static::lazy_static! {
 pub struct Indexer<T, U, Db>
 where
     T: HoprIndexerRpcOperations + Send + 'static,
-    U: ChainLogHandler + Send + 'static,
+    U: ChainLogHandler + Clone + Send + 'static,
     Db: HoprDbGeneralModelOperations + Clone + Send + Sync + 'static,
 {
     rpc: Option<T>,
     db_processor: Option<U>,
```

Comment:
@coderabbitai Please create an issue to rewrite these fields in terms of the `Once` object, but do not assign the issue to anyone.